### PR TITLE
Feature/CU 8685en995 Shared App Time Limits

### PR DESCRIPTION
**Pull request by Aleyum**
- Added isExpired field to  `ContentTypes`

### CODE REVIEW

File: Assets/MXR.SDK/Runtime/Types/ContentTypes.cs [hunk 1]
```diff
@@ -208,6 +208,11 @@ public class RuntimeApp : Content {
         /// </summary>
         public string packageName;
 
+        /// <summary>
+        /// Whether the application has expired or not
```
Comment (johnmanagexr) [inline Assets/MXR.SDK/Runtime/Types/ContentTypes.cs:212]: This comment mostly just reiterates the name of the field. Can you make this clear to someone with less context who is looking at our SDK?

Maybe something like: 

"Whether the application is currently in an expired state, which happens on shared applications when they reach the end of their shared time limit."

**Review by johnmanagexr — CHANGES_REQUESTED** [review]

**Review by johnmanagexr — APPROVED** [review]

**Review by adrenak — APPROVED** [review]

**Comment by adrenak** [comment]
Raised some new points for a couple of util methods

**Review by adrenak — CHANGES_REQUESTED** [review]

File: Assets/MXR.SDK/Runtime/Android/Utils/MXRAndroidUtils.Apps.cs [hunk 2]
```diff
@@ -150,5 +150,21 @@ public static partial class MXRAndroidUtils {
         "This method may be removed in the future.")]
         public static void RequestManageAllFilesPermission() =>
             RequestManageAppAllFilesAccessPermission();
+
+        public static string EnumToString<TEnum>(TEnum enumValue) where TEnum : Enum {
```
[the remark this answers is not in the document]
Comment (johnmanagexr) [reply Assets/MXR.SDK/Runtime/Android/Utils/MXRAndroidUtils.Apps.cs:154]: This is resolved without a comment - is there a reason for this?

If you chatted about this offline, it's worthwhile to follow up on a comment here as well!

File: Assets/MXR.SDK/Runtime/Android/Utils/MXRAndroidUtils.Apps.cs [hunk 3]
```diff
@@ -150,5 +150,21 @@ public static partial class MXRAndroidUtils {
...
+            return enumValue.ToString();
+        }
+
+        public static string MillisecondsToDate(long dateInMS) {
```
[the remark this answers is not in the document]
Comment (johnmanagexr) [reply Assets/MXR.SDK/Runtime/Android/Utils/MXRAndroidUtils.Apps.cs:158]: Same here

**Review by johnmanagexr — CHANGES_REQUESTED** [review]
Just following up on loose ends, otherwise looks good

File: Assets/MXR.SDK/Runtime/Android/Utils/MXRAndroidUtils.Apps.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (Aleyum) [reply Assets/MXR.SDK/Runtime/Android/Utils/MXRAndroidUtils.Apps.cs]: Resolved via DMs, just forgot to remove the unused method.

File: Assets/MXR.SDK/Runtime/Android/Utils/MXRAndroidUtils.Apps.cs (hunk 3 above)
[the remark this answers is not in the document]
Comment (Aleyum) [reply Assets/MXR.SDK/Runtime/Android/Utils/MXRAndroidUtils.Apps.cs:154]: Again, moved this but forgot to remove the original function

**Review by adrenak — APPROVED** [review]

**Review by johnmanagexr — APPROVED** [review]

**Pull request merged**